src/__tests__/Field.spec.js

```diff
@@ -606,61 +606,61 @@ const describeField = (name, structure, combineReducers, expect) => {
     // ----------------------------------------------
     // Uncomment this to confirm that #1024 is fixed.
     // ----------------------------------------------
```

Comment:
@erikras Oh, I think that syncError is not related to props, so when I avoid the unnecessary rerender, the test failed.